frontend/src/pages/home.tsx

```diff
@@ -88,8 +129,121 @@ const Home: FC = () => {
         </div>
       </section>
 
-      <section>
-        <Tabcordian data={t('tabs', { returnObjects: true })} />
+      <section className='py-10 md:flex md:flex-col md:items-center'>
+
+        <TabContext value={value}>
+          <TabList
+            variant='scrollable'
+            scrollButtons='auto'
+            onChange={handleChange}
+            className='z-20 flex elevation-4 center -mx-4 md:w-fit md:rounded-2xl bg-white text-button-background md:mx-4 md:px-24'
+            classes={{
+              scrollableX: "mx-4",
+              flexContainer: "h4 box-border flex cursor-pointer capitalize appearance-none rounded-xl focus:outline-none "
+            }}
+            visibleScrollbar={true}
+            >
+              {tabData.map((item, i) => (
+                <Tab
+                className='h4 text-base box-border flex cursor-pointer capitalize appearance-none pt-4 pb-1 px-12 rounded-xl focus:outline-none' 
+                key={item.title} value={item.title} label={item.title} />
+                ))}
+          </TabList>
+          <div className="relative md:w-full md:h-full -z-10 -mx-4 md:rounded-2xl px-4 py-6 md:relative -mt-4 md:-mt-6 bg-gray-surface md:px-24 md:py-16 lg:px-24">
+            {tabData.map((item, i) => (
+              <TabPanel className='py-4 px-1' key={item.title} value={item.title}>
+                <div className='flex flex-col md:flex-row md:space-x-6 elevation-1 rounded'>
+                <div className="rounded bg-white px-10  elevation-1 md:w-2/5 md:grow" >
+                    <h3 className="h5 md:font-medium md:font-display md:text-4xl text-aqua-dark flex items-center py-8">
+                      {item.heading}
+                    </h3>
+                    <div className="justify-center border-y-2 py-8">
+                      {item.description.map((desc, i) => (
+                        <p key={i}>
```

Comment:
Index shouldn't be used as a key here.
https://react.dev/learn/rendering-lists#why-does-react-need-keys